requirements.txt

```diff
@@ -1 +1,2 @@
 Django>=1.7
+djaodjin-deployutils==0.5.6
```

Comment:
djaodjin-multitier is installed as a Django App inside a Django project. As such the requirements.txt shouldn't include a strict version number (`==`) but minimum/compatible requirements.